UITests/ClearPrivateDataTests.swift

```diff
@@ -100,18 +100,19 @@ class ClearPrivateDataTests: KIFTestCase, UITextFieldDelegate {
         
         // Only one will be found -- we collapse by domain.
         anyDomainsExistOnTopSites(dispDomains, fulldomains: fullDomains)
-
         BrowserUtils.clearPrivateData([BrowserUtils.Clearable.History], swipe: false, tester: tester())
-        EarlGrey.select(elementWithMatcher: grey_accessibilityLabel(urls[0].title))
+    
+        // Tap bookmark and come back to refresh DOM
+        EarlGrey.select(elementWithMatcher:  grey_accessibilityID("HomePanels.Bookmarks")).perform(grey_tap())
+        EarlGrey.select(elementWithMatcher:  grey_accessibilityID("HomePanels.TopSites")).perform(grey_tap())
+        
+        let displayed = urls[0].dispDomain.replacingOccurrences(of: ".", with: " ")
+        EarlGrey.select(elementWithMatcher: grey_accessibilityLabel(displayed))
```

Comment:
so this happens only with ip addresses.  For example if we go to 127.0.0.1, in the topsitepanel, it shows up as "127 0 0" which is odd, I think I filed a bug for it, but it's buried somewhere deep in bugzilla.